tests/minigraph/test_masked_services.py

```diff
@@ -41,19 +41,27 @@ def change_service_state(duthost, service, enable):
 @pytest.mark.disable_loganalyzer
 def test_masked_services(duthosts, rand_one_dut_hostname):
     """
-    @summary: This test case will mask telemetry service, then test load_minigraph and check its success
+    @summary: This test case will mask telemetry/gnmi service, then test load_minigraph and check its success
     """
     duthost = duthosts[rand_one_dut_hostname]
-    test_service = "telemetry"
-    logging.info("Bringing down telemetry service")
-    service_status = duthost.critical_process_status("telemetry")
+    containers_states, succeeded = duthost.get_feature_status()
```

Comment:
>get_feature_status [](http://example.com/codeflow?start=43&length=18)

Instead of using running status of feature, is it better to use config of feature?

For example, if telemetry feature is configured as enabled, but crashed, fail this testcase instead of fallback to gnmi feature.

Reply:
Feature config is not correct, for example, telemetry is enabled, but there's no telemetry service.
I will use docker image to check feature status, if there's telemetry docker image, we should telemetry service.